modules/nf-core/fastcov/environment.yml

```diff
@@ -0,0 +1,6 @@
+name: fastcov
+channels:
+  - conda-forge
+  - bioconda
+dependencies:
+  - fastcov
```

Comment:
Which channel is this coming from? Need to add.

Reply:
its from bioconda, I added fastcov to the bioconda registry :)

Reply:
Great, then add bioconda:: and specify the version number please.